client/src/components/GroupHeader/GroupHeaderContainer.js

```diff
@@ -42,11 +17,25 @@ export default class GroupHeaderContainer extends Component {
 
 }
 	render() {
+		const { id } = this.props.match.params
+
 		return (
-			<div className="group-header-container"> 
-				<GroupHeaderBanner />
-				<GroupHeaderMenu member={this.state.member}/>
-			</div>
+			<Query query={GET_GROUP} variables={{ id }}>
```

Comment:
I think your tab size is too big, it makes this hard to read.

Reply:
sorry about that, i'll switch to using spaces now